src/profile-logic/symbolication.js

```diff
@@ -404,6 +404,16 @@ export function applySymbolicationStep(
     IndexIntoFuncTable
   > = new Map();
 
+  // We want to group frames into funcs, and give each func a name.
+  // We group frames to the same func if the addresses for these frames resolve
+  // to the same funcAddress.
+  // We obtain the funcAddress from the symbolication information in resultsForLib:
```

Comment:
Ah, so this is where my confusion came from on "choosing a func address". I didn't realize that the symbolication step actually gave us the func address.